setup.php

```diff
@@ -139,6 +142,12 @@ function plugin_init_formcreator() {
    if ($plugin->isInstalled('formcreator') && $plugin->isActivated('formcreator')) {
       spl_autoload_register('plugin_formcreator_autoload');
 
+      /*
+       * Plugin configuration parameters -
+       * set the content of the global $PFC_CONFIG variable according to the current session identifier
+       */
+      PluginFormcreatorEntityconfig::loadConfiguration();
```

Comment:
unit tests broken because of this call. 

2 solutions: 
- use a new argument --as-user like Fusion Inventory does
- protect this call with a if() checking that a user is logged in. This may help for anonymous users when they use a form.

Reply:
Ok. I will check if a user is logged-in inside the function